mne/viz/utils.py

```diff
@@ -1164,6 +1165,8 @@ def _fake_click(fig, ax, point, xform='ax', button=1, kind='press'):
         x, y = ax.transAxes.transform_point(point)
     elif xform == 'data':
         x, y = ax.transData.transform_point(point)
+    elif xform is 'pix':
```

Comment:
don't use `is` with string comparison, use `==`

Reply:
ah, yes, I'll change that.